corehq/apps/reports/static/reports/js/tabular.js

```diff
@@ -69,4 +74,23 @@ hqDefine("reports/js/tabular", function() {
             renderPage(initialPageData("js_options").slug, initialPageData("report_table_js_options"));
```

Comment:
I hate to bring this up here, and I couldn't find a guideline about it in the [js-guide](https://github.com/dimagi/js-guide), but didn't we say we were going to try to not have document on-ready handlers in the same module that returns a module value?

In this case the thing that made me notice it was the comment at the top about `reportTables`. I noticed that the two returned functions access but do not initialize `reportTables`, which made me look deeper to see how it is initialized.